Moderate-Background-Image/README.md

```diff
@@ -0,0 +1,34 @@
+OpenTok.js Moderate Background Replacement
+=======================
+
+This sample application shows how to add image background replacement with the Vonage
+Video APIs. It is very similar to the [Basic Video Chat](../Basic%20Video%20Chat/) example, but it adds a background image to the publisher. See [Vonage ML Transformers](https://vonage.github.io/ml-transformers-docs/) library for more information.
+
+## Demo
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/fork/github/opentok/opentok-web-samples/tree/main/Moderate-Background-Blur)
```

Comment:
Please update the stackblitz link to point to this sample not the background blur sample

Reply:
Thanks @devwithzachary, updated